src/transformers/__init__.py

```diff
@@ -178,6 +178,14 @@
     ],
     "models.bloom": ["BLOOM_PRETRAINED_CONFIG_ARCHIVE_MAP", "BloomConfig"],
     "models.bort": [],
+    "models.bridgetower": [
+        "BRIDGETOWER_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "BridgeTowerConfig",
+        "BridgeTowerImageProcessor",
```

Comment:
The image processor should be under an `is_vision_available` (as it relies on the Pillow library), see any other image processor as an example